integration-cli/docker_cli_run_test.go

```diff
@@ -2268,11 +2473,14 @@ func (s *DockerSuite) TestRunSetDefaultRestartPolicy(c *check.C) {
 }
 
 func (s *DockerSuite) TestRunRestartMaxRetries(c *check.C) {
-	testRequires(c, DaemonIsLinux)
 	out, _ := dockerCmd(c, "run", "-d", "--restart=on-failure:3", "busybox", "false")
+	timeout := 10 * time.Second
```

Comment:
this value is translated to `time.Duration` inside the `waitInspect` function. I think the problem with the flaky test is that the double translation causes that value to not be 10/45 seconds, I've seen similar issues in the past